src/lib/features/feature-toggle/feature-toggle-service.ts

```diff
@@ -1235,6 +1235,18 @@ class FeatureToggleService {
         }
     }
 
+    private async validateActiveProject(projectId: string) {
```

Comment:
This form sounds like "check something in active project"
```suggestion
    private async validateProjectIsActive(projectId: string) {
```

Reply:
will address in another PR cause we have 2 similar method in project and feature service